be/src/formats/parquet/column_chunk_reader.h

```diff
@@ -131,6 +134,11 @@ class ColumnChunkReader {
 
     Status _read_and_decompress_page_data(uint32_t compressed_size, uint32_t uncompressed_size, bool is_compressed);
 
+    bool _no_null() {
```

Comment:
_has_null() better?

Reply:
i think _no_null() is more direct, because the statistics is optional, _has_null may mean no statistics or really has_null